src/gabc/gabc-glyphs-determination.c

```diff
@@ -774,31 +881,86 @@ gregorio_glyph *gabc_det_glyphs_from_notes(gregorio_note *current_note,
             continue;
         }
 
-        /*
-         * first we do what must be done with liquescentia
-         */
-        if (is_initio_debilis(current_note->u.note.liquescentia)) {
-            /*
-             * meaning that the note is an initio debilis, maybe more
-             */
+        /* first we do what must be done with liquescentia */
+        head_liquescentia = current_note->u.note.liquescentia
+                & (L_INITIO_DEBILIS | L_FUSED);
+        if (head_liquescentia) {
+            /* initio debilis or fused */
             if (current_glyph_type != G_UNDETERMINED) {
-                /*
-                 * if it is not the first glyph
-                 */
+                /* if it is not the first glyph */
                 close_glyph(&last_glyph, current_glyph_type,
                         &current_glyph_first_note,
                         liquescentia, current_note->previous);
                 current_glyph_type = G_UNDETERMINED;
             }
-            liquescentia = L_INITIO_DEBILIS;
+            liquescentia = head_liquescentia;
+        }
+
+        if (autofuse) {
+            if (current_glyph_first_note == current_note) {
+                if (first_autofused_note) {
```

Comment:
Testing the oriscus feature, I realize I have a warning here:

```
gabc/gabc-glyphs-determination.c: In function ‘gabc_det_glyphs_from_notes’:
gabc/gabc-glyphs-determination.c:895:20: warning: ‘first_autofused_note’ may be used uninitialized in this function [-Wmaybe-uninitialized]
                 if (first_autofused_note) {
```

(the line number doesn't correspond)